### ISSUE TITLE
Not all characters or icons displaying correctly in Windows XP

### ISSUE THREAD

**Issue by EdDixon** [learningequality/ka-lite#3587]
An issue was discovered during testing of the 13.1 branch on 4/30/2015 that may affect WIndows XP users. However, In discussion with other developers it was decided this is probably not an installer related issue so posting here. The red and green symbols showing correct or incorrect responses to assessment items do not render in windows XP and other symbols or characters may also be affected by this issue. Where expected would be X's or Check Marks we have character place boxes in the assessment dialog as show below. This was tested on several different OS's (MAC, Linux, Windows 7) and was only replicated in Windows XP. 
![2015-04-30_11-02-45](https://cloud.githubusercontent.com/assets/4349806/7425395/3fe6e02e-ef6d-11e4-9129-771db6ccf9d2.png)

**Comment by MCGallaspy** [comment]
This is probably an issue with the available fonts. Is it worthwhile to change our app to use images instead of unicode characters? Or choose unicode characters that have broader support? @rtibbles

**Comment by aronasorman** [comment]
Fixed in 0.14.

**Comment by aronasorman** [comment]
Not sure, but @EdDixon indicated in the dev room that he's no longer encountering this on his XP VM.

**Comment by EdDixon** [comment]
It is definitely back in 0.15.0? :(

**Comment by radinamatic** [comment]
@rtibbles Could these be the missing .woff2 chars (#4911)? 

0.16 Alpha4 on both Chromium & FF (Ubuntu):

![learn numbers to 100 ka lite - chromium_003](https://cloud.githubusercontent.com/assets/1457929/13340377/1e616504-dc30-11e5-8adb-92c99904d103.png)

But they do appear on build 150 on W7.

Mystery...

**Comment by radinamatic** [comment]
My bad... Screenshot above is from non-logged in user, re-checked and chars appear fine on both Win & Ubuntu. This looks fixed!

**Comment by radinamatic** [comment]
I spoke too soon... This does seem to be (still) an XP specific issue! :(

FF
![xp](https://cloud.githubusercontent.com/assets/1457929/13482121/defaa51e-e0ea-11e5-9212-ca2ba88c281e.png)

Chrome
![xp2](https://cloud.githubusercontent.com/assets/1457929/13482215/ae04d85c-e0eb-11e5-99cd-145e32936950.png)

@rtibbles @MCGallaspy

**Comment by radinamatic** [comment]
Found also some .svg images not showing correctly in both FF & Chrome on XP (build 167) despite being present in the `.kalite/content` folder. Documenting here so I can re-test once the fix is merged.

![ie8 - winxp 0 16 build 167 running - oracle vm virtualbox_259](https://cloud.githubusercontent.com/assets/1457929/13570432/85ffdc18-e46e-11e5-9e4d-bb58a6cf3698.png)

![ie8 - winxp 0 16 build 167 running - oracle vm virtualbox_260](https://cloud.githubusercontent.com/assets/1457929/13570463/a527dd2a-e46e-11e5-9589-53e9562f3820.png)

**Comment by MCGallaspy** [comment]
@radinamatic if you're trying to test the changes in #4965, it won't be reflected from the build server installers until it's merged into `0.16.x` -- so you'll have to install KA Lite from source. Or if necessary I can temporarily change the branch they're built from.

**Comment by rtibbles** [comment]
Fixed in #4965
